config/sample-config.yaml

```diff
@@ -1304,16 +1308,16 @@ frontend:
         - 1: Egress (outgoing traffic, from the node observation point) +
         - 2: Inner (with the same source and destination node)
     - name: IfDirections
-      type: number
+      type: number array
```

Comment:
in another PR ( https://github.com/netobserv/network-observability-operator/pull/999/files#diff-1fc45b6b65862012ab2d4a19e8f92ce31c1801f0ed7c464c13ac3f0dc5a3b756R1268 - which you can review btw ;-)  ), I'm doing the same kind of thing, except I use a new field name to avoid breaking something in the console. So, I guess it means I can rollback that part.
Just, I used the brackets convention to describe an array, like `number[]` ; wdyt, I'd say it's sufficiently known convention so we can use it?

Reply:
I can change the `array` keyword to `[]` indeed :ok_hand:

Reply:
an advantage of `[]` is that it takes less space here https://docs.openshift.com/container-platform/4.17/observability/network_observability/json-flows-format-reference.html

Reply:
although in the API reference 
![image](https://github.com/user-attachments/assets/044a5f1c-af83-41b7-804d-85206a304c1b)
it's "array (string)", so yet something else. Honestly, I don't care too much :-) It's just that `string[]` felt more natural to me, of course I have my developer bias

Reply:
https://github.com/netobserv/network-observability-console-plugin/pull/681/commits/93b52ddc65074f88f18c2defbeb619d2548692c8